docs/content/components/field.mdx

```diff
@@ -7,17 +7,17 @@ title: Field
 ## Description
 
 With the Field component you can add a HTML `<label>` and `<input>` element to your form.
-The variant can be added with the variant prop. The default variant is `default`.
 
 ## Properties
 
-| Property             | Type                                                                                       | Default   |
-| :------------------- | :----------------------------------------------------------------------------------------- | :-------- |
-| `variant` (optional) | `string`                                                                                   | `default` |
```

Comment:
Isn't it a bit odd that we don't have a variant here anymore?

Reply:
Yes it is. But how would you handle this? It's just an addition of Label + Input and they have their fitting variants. I can't think of a suitable situation for a field variant.